main.cpp

```diff
@@ -0,0 +1,92 @@
+#include <iostream>
+#include <vector>
+
+using namespace std;
+
+class rs_m
+{
+public:
+    vector<double> r; 
+
+    void setResort(vector<double> r)
+    {
+        this->r = r;
+    }
+
+    //getting the size of the resort
+    double GetResortSize() 

```

Comment:
I believe c++ function naming standard is lower camel case, so the function name should begin with a lower "g".
Also, I think that returning `int` in this case would be better, since you return a discrete number, not a real one?